src/server.js

```diff
@@ -7,7 +7,7 @@ fastify.register(require('fastify-cors'))
 fastify.register(require('fastify-postgres'), {
   connectionString: process.env.DATABASE_URL,
 })
-fastify.register(require('./routes'))
+fastify.register(require('./routes/userRoutes/userRoutes'))
```

Comment:
Since we are decoupling the authentication logic from the users route, we can actually just require the userRoutes folder and have it default to index.js